tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleProjectProvider.java

```diff
@@ -32,7 +31,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import org.eclipse.buildship.core.configuration.GradleProjectNature;
+import org.eclipse.buildship.core.internal.configuration.GradleProjectNature;
```

Comment:
Do we need this internal class?

Reply:
When we create a new project, we will invoke synchronization.  When it is synchronized Buildship will call their new ProjectConfigurator API.  That is where we should add our nature, not when they add their nature.  I think it would be better if used ProjectConfigurator instead of ProjectCreatedEvent.